src/Uno.UI.Runtime.Skia.Wpf/Islands/UnoXamlHostBase.Focus.cs

```diff
@@ -8,12 +8,16 @@
 using WF = Windows.Foundation;
 using WUX = Windows.UI.Xaml;
 
+//TODO:MZ: We need to make sure that when the UnoXamlHost loses focus, focus is changed in the XamlRoot as well,
```

Comment:
Let's create an epic for tracking those.

Reply:
#8978